src/lib/feed-run.ts

```diff
@@ -17,7 +17,7 @@ const eventFeedTimers: any[] = [];
 const publishStats:any = {};
 
 const feedRun = async (options: ManageFeedPublishOptions, optionsSource: any) => {
-  const { helpExamples } = options
+  const { helpExamples, quite } = options
```

Comment:
is this supposed to be "quite" or should it be "quiet" ?

Reply:
Ah.. it is quite a misspelling :) 

Fixed it quietly...